modules/machine-api-overview.adoc

```diff
@@ -5,6 +5,8 @@
 // * machine_management/creating_machinesets/creating-machineset-gcp.adoc
 // * machine_management/creating_machinesets/creating-machineset-osp.adoc
 // * machine_management/creating_machinesets/creating-machineset-vsphere.adoc
+// * windows_container_workloads/creating_windows_machinesets/creating-windows-machineset-aws.adoc
+// * windows_container_workloads/creating_windows_machinesets/creating-windows-machineset-azure.adoc
 
```

Comment:
In the section below, s/MachineHealthChecks/machine health checks (in the NOTE box)

Reply:
Also,    s/MachineSet/machine set